internal/rest/avi_obj_vsvip.go

```diff
@@ -77,8 +77,9 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, vsCach
 
 		// handling static IP and networkName (infraSetting) updates.
 		vip := &avimodels.Vip{
-			VipID:          &vipId,
-			AutoAllocateIP: &autoAllocate,
+			VipID:                  &vipId,
+			AutoAllocateIP:         &autoAllocate,
+			AutoAllocateFloatingIP: &vsvip_meta.EnablePublicIP,
```

Comment:
Can you make sure this works with Essentials license, without the public IP (for regressions)

Reply:
done, works fine